packages/blade/src/components/Tour/_decisions/decisions.md

```diff
@@ -0,0 +1,766 @@
+- [React Native Specifics](#react-native-specifics)
+- [Multiple Tour Flows](#multiple-tour-flows)
+  - [Approach-1: Prefix `id` to avoid conflicts](#approach-1-prefix-id-to-avoid-conflicts)
+  - [Approach-2: Use `TourStep` to avoid conflicts](#approach-2-use-tourstep-to-avoid-conflicts)
+  - [Technical Challenge in React Native](#technical-challenge-in-react-native)
+- [API Design Challenges](#api-design-challenges)
+  - [Approach: Let Consumer Compose `Popover`](#approach-let-consumer-compose-popover)
+    - [**1. Not possible to collocate the tour flows.**](#1-not-possible-to-collocate-the-tour-flows)
+    - [**2. How will we have two different tour flows in the same page?**](#2-how-will-we-have-two-different-tour-flows-in-the-same-page)
+    - [3. Segmentation / Lack of control](#3-segmentation--lack-of-control)
+    - [4. Not possible to maintain a consistent API / Implementation between web \& native](#4-not-possible-to-maintain-a-consistent-api--implementation-between-web--native)
+    - [Conclusion](#conclusion)
+- [MoMs](#moms)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=63871%3A13263&mode=dev) to all variants of the Tour component
+
+## Features
+
+- [x] **Tour:** A guided tour with multiple steps
+- [x] **Masking:** Mask the rest of the page except the highlighted element.
+
+## Basic Usage Structure
+
+With Tour component, you can specify an array of steps with a `name` prop for each step, and then use the `TourStep` component to wrap the element that needs to be highlighted with the same `name` prop.
+
+The Tour component internally handles everything for your
+
+- Masking of the highlighted component
+- Internal state management of `activeStep`
+- Navigation between steps
+
+> Check the full example on [Usage](#usage) section below
+
+```jsx
+const steps = [
+  {
+    name: 'step-1',
+    title: 'Step 1',
+    content: () => <Text>Some content for step 1</Text>,
+    footer: () => <Text>Footer for step 1</Text>,
+  },
+  // ...more steps
+];
+
+<Tour steps={steps}>
+  // ... more jsx
+  <TourStep name="step-1">
+    <Box>I'll be highlighted</Box>
+  </TourStep>
+</Tour>;
+```
+
+## API
+
+### `Tour` API
+
+The `Tour` component is used to render a guided tour with multiple steps.
+Each tour `step` will have a subset of props from the `Popover` component, with extra props for the tour related logic.
+
+> Check the Popover API Decisions [here](https://github.com/razorpay/blade/blob/master/packages/blade/src/components/Popover/_decisions/decisions.md)
+
+```jsx
+// Step will have similar props as the Popover component, With extra Tour related props.
+type Step = {
+  /**
+   * Unique identifier for the tour step
+   */
+  name: string;
+  /**
+   * Content of the Popover
+   */
+  content: ({ gotToNext, goToPrev, stopTour, activeStep, totalSteps }) => React.ReactNode;
```

Comment:
> Shouldn't description be a mandatory prop with an optional

So there's actually no `description` prop on Figma. 

This description text is there for designer's ease of use, so that each time they use the Popover they don't have to pull in a Text component to put on the slot. 
Designers can remove this default description text add anything they want. So on dev it's just the content `children`